# 
# Submitted:
#
# Robert Schwebel, 2004-11-03
#
# Error: 
#
# - use autotools with defined versions
# - generate archive files
#
# State:
# 
# leave locally
#

diff -x CVS -urN DamageExt/autogen-rs.sh DamageExt-ptx/autogen-rs.sh
--- DamageExt/autogen-rs.sh	1970-01-01 01:00:00.000000000 +0100
+++ DamageExt-ptx/autogen-rs.sh	2004-11-03 07:40:33.000000000 +0100
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+aclocal-1.7 -I /usr/share/aclocal/
+#autoheader
+autoconf
+automake-1.7 --add-missing --copy
+
diff -x CVS -urN DamageExt/configure.ac DamageExt-ptx/configure.ac
--- DamageExt/configure.ac	2004-11-03 08:35:15.000000000 +0100
+++ DamageExt-ptx/configure.ac	2004-11-03 08:10:27.000000000 +0100
@@ -25,7 +25,7 @@
 
 AC_PREREQ([2.57])
 AC_INIT([damageext], [1.0], [keithp@keithp.com], damageext)
-AM_INIT_AUTOMAKE([dist-bzip2])
+AM_INIT_AUTOMAKE(DamageExt, 20041103-1, [dist-bzip2])
 AM_MAINTAINER_MODE
 AC_CONFIG_AUX_DIR(.)
 
diff -x CVS -urN DamageExt/Makefile.am DamageExt-ptx/Makefile.am
--- DamageExt/Makefile.am	2004-11-03 08:35:15.000000000 +0100
+++ DamageExt-ptx/Makefile.am	2004-11-03 08:10:00.000000000 +0100
@@ -33,5 +33,5 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = damageext.pc
 
-EXTRA_DIST = autogen.sh damageext.pc.in $(damageextdoc_DATA)
+EXTRA_DIST = autogen.sh autogen-rs.sh damageext.pc.in $(damageextdoc_DATA)
 
